docs/api/apiv3/endpoints/schemas.apib

```diff
@@ -49,10 +49,17 @@ Only one of the links (`allowedValues`, `allowedValuesSchemas`) will exist for a
 | regularExpression | The value of the property must match the given regular expression (if not null)    | String   | null    |
 | required          | If true this property is not nullable                                              | Boolean  | true    |
 | writable          | If false it is not allowed to **change** the properties value                      | Boolean  | true    |
+| location          | If present, contains a reference to the location of the property in the JSON       | String   |         |
 
 All of the above properties that do not have a default value *must* be present in the schema.
 For properties that have a default value, the client can assume the default value, if the property is missing.
 
+### Location property
+
+The location property gives a hint as to where to find the property if it is not contained in the set of local attributes.
+For example, for linked properties, the location attribute of the schema will contain `_links`. The property will then be found in the resource
+under the path `_links.propertyName`.
+
 Note that regular expressions used in the API follow the rules of [Ruby Regular Expressions](http://www.ruby-doc.org/core-2.2.6/Regexp.html).
```

Comment:
This should either get its own heading now or be placed above the `Location property` section